scripts/distribution/builder.Dockerfile

```diff
@@ -64,17 +64,19 @@ EXPOSE 8099
 EXPOSE 9999
 # GRPC
 EXPOSE 10000
-ENV RPC_SERVER_ADDR=0.0.0.0
-ENV MODE=basic
-ENV BOOTSTRAP_FIRST_NODE=bootstrap."${environment}":8888
-ENV DATA_DIR=/var/lib/concordium/data
-ENV CONFIG_DIR=/var/lib/concordium/config
-ENV EXTRA_ARGS="--no-dnssec"
-ENV NODE_URL=localhost:10000
-ENV COLLECTORD_URL=https://dashboard."${environment}"/nodes/post
-ENV GRPC_HOST=http://localhost:10000
-ENV DISTRIBUTION_CLIENT=true
-ENV ENABLE_TERM_HANDLER=true
+    
+# Concordium Node configuration    
+ENV CONCORDIUM_NODE_RPC_SERVER_ADDR=0.0.0.0
+ENV CONCORDIUM_NODE_DATA_DIR=/var/lib/concordium/data
+ENV CONCORDIUM_NODE_CONFIG_DIR=/var/lib/concordium/config
+ENV CONCORDIUM_NODE_GRPC_HOST=http://localhost:10000
+ENV CONCORDIUM_NODE_CONNECTION_BOOSTRAP_NODES=bootstrap.testnet.concordium.com:8888
+ENV CONCORDIUM_NODE_CONNECTION_NO_DNSSEC=1
+ENV CONCORDIUM_NODE_CONNECTION_BOOSTRAP_NODES=bootstrap."${environment}":8888
+    
+# Concordium Node Collector configuration
+ENV CONCORDIUM_NODE_COLLECTOR_URL=https://dashboard."${environment}"/nodes/post
```

Comment:
After #73 is merged this needs to be updated as well.

Reply:
Git will probably help us there. :-)